README.md

```diff
@@ -32,13 +40,15 @@ To use the CSS animations and effects in your project, simply download or clone
 
 <br> 
 
-<p align="center"><a href="#snippets">^^ Back to Top ^^</a></p>
+<p align="center"><a href="#snippets">^^ Back to Snippets Top ^^</a></p>
 
 ## Contributing
 
-We welcome contributions to the CSS-Is-Fun project!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
+We welcome contributions .Welcome to CSS is Fun on GitHub! We're excited for your contributions, big or small. Join our inclusive community and work on various CSS tasks to improve design and functionality. Let's collaborate and make something great together!🚀 For detailed instructions on how to contribute, please refer to the CONTRIBUTING.md file.
```

Comment:
Here the fullstop is with Welcome like this` .Welcome` 

and `welcome` seems repetitive 

Make it like this

```diff
+ We encourage contributions. Welcome
- We welcome contributions .Welcome
```